src/scripts/add-internet-gateways-to-default-vpcs-route-tables.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+function query() {
+  ret_val=$(psql postgres://$username:$password@localhost:5432/iasql -AXqtc "$1")
+}
+
+query "SELECT iasql_install('aws_vpc');"
+
+query "SELECT ig.id, vpc.id
+       FROM internet_gateway ig
+          INNER JOIN vpc on vpc.id = ig.vpc_id
+       WHERE vpc.is_default = true;"
+ids=$ret_val
+
+query "SELECT iasql_begin();"
+for id in $ids; do
+  IFS='|' read -ra splitarr <<< "$id"
+  ig_id=${splitarr[0]}
+  vpc_id=${splitarr[1]}
```

Comment:
Cool command. Want to make it more cool? :D
![image](https://user-images.githubusercontent.com/5357435/213779218-1603967f-07b3-4cfe-8cff-07c8a459aa75.png)

```suggestion
  IFS='|' read -r ig_id vpc_id <<< "$id"
```